README.md

```diff
@@ -129,14 +126,6 @@ I thank my partiners in learning and coding for better success in this program.
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 How did this man fix the linters ?
 
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Ouch! this is close but please make sure to follow the Microverse template in the FAQ section here because whenever you create a question you should also provide an answer right below the question as shown in the template, so please provide an answer to the question "How did this man fix the linters ?" 😊